index.html

```diff
@@ -33,13 +39,128 @@
                   ? `https://github.com/prefuse/Flare/tree/master/flare/src/${n.ancestors().reverse().map(d => d.data.name).join("/")}`
                   : `https://github.com/prefuse/Flare/blob/master/flare/src/${n.ancestors().reverse().map(d => d.data.name).join("/")}.as`,
                 */
-                width: 1152,
-                height: 1152
+                width: mainWidth,
+                height: mainHeight
             });
 
             document.querySelector("#sunburstContainer")
                 .appendChild(chart);
-        });
+
+            
+            //PNG EXPORT FUNCTIONALITY
+            // Set-up the export button
+            d3.select('#saveButton').on('click', function(){
+              let svgString = getSVGString(document.getElementById('theImage'));
```

Comment:
How does it magically know that `#theImage` is the right selector? Is this a D3 standard I don't know about?

Reply:
I added 'theImage' as the svg ID in sunburst.d3.js line 74:  `.attr("id", 'theImage');`
I just did this because I couldn't work out a neater way to select the svg - the other things I tried weren't working. If there's a better/easier way I'd love to know :)